### DISCUSSION TITLE
HOw to start dpgen exploration from an existing frozen_model.db file

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/dpgen#409] [Q&A]
Hi,

I have a trained model for a system, which works ok but at high temperature it shows some issue. I am wondering if I could perform further training using dpgen using the available frozen_model.pb files. It would be of a great help.

I have another question regarding training step/stop_batch should be chosen in param.json file.  

Thanks,
Mayank

**Comment by njzjz** [comment]
In fact, you usually need 4 models in order to calculate model deviation. See our paper.
